### ISSUE TITLE
Buddybuild crash report on build #210

### ISSUE THREAD

**Issue by sarentz** [mozilla-mobile/prox#202]
Buddybuild detected a crash from bnicholson@mozilla.com
`Prox/AppDelegate.swift` line 9`
```swift
import UIKit
import Firebase

@UIApplicationMain
class AppDelegate: UIResponder, UIApplicationDelegate { <-- CRASHED

    var window: UIWindow?
    var placeCarouselViewController: PlaceCarouselViewController?

    private var authorizedUser: FIRUser?
```

[View File](https://github.com/mozilla-mobile/prox/blob/8c23072fed174ffb5baf07bffa842c570b6a7f5a/Prox/AppDelegate.swift#L9) | [View Full Crash Details](https://dashboard.buddybuild.com/apps/57f51f8bcaa70a01005412ee/crashreport/af50d5c3237ab3d9c19cdf790152298d)

**Comment by sarentz** [comment]
Crash report from bnicholson@mozilla.com

**Comment by fluffyemily** [comment]
build #210 has a known crash bug that was fixed in build #253. Upgrade to =
#253 and the crash will go away
